notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: wcreset\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Option \n",
+    "        \"wreset\"  resets  the  graph  limits  to  those specified by the\n",
+    "        \u000exmin\u000f, \u000exmax\u000f, \u000eymin\u000f, \u000eymax\u000f parameters whenever  a  new  spectrum  is\n",
+    "        plotted.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: flip\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "The  \"flip\" option selects that initially the spectra\n",
+    "        be plotted with decreasing  wavelengths. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: auto\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Option  \"auto\"\n",
+    "        automatically  replots  the  graph  whenever  changes  are made.\n",
+    "        Otherwise the graph is replotted with  keystrokes  'c'  or  'r'."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: zero\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Option  \"zero\"  makes  the initial minimum y of the graphs occur\n",
+    "        at zero.  Otherwise the limits are set  automatically  from  the\n",
+    "        range  of  the  data  or  the  \u000eymin\u000f  parameter. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initial limits of graph (xmim,xmax,ymin,ymax)\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    " The  default  limits  for  the initial graph.  If INDEF then the\n",
+    "        limit is determined from the range of  the  data  (autoscaling).\n",
+    "        These  values  can  be changed interactively with 'w' window key\n",
+    "        options or the cursor commands \":/xwindow\" and \":/ywindow\"  (see\n",
+    "        \u000egtools\u000f).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Measurements over interactively-specified spectral regions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many science use cases depend on measurements of parts of a spectrum like spectral lines, which are most easily specified interactively in \"quick look\" situations.  Specviz should support this use case by:\n",
+    "\n",
+    "1. For notebooks, providing an easy way to extract SpectralRegion objects from selections made in the specviz UI. This would most straightforwardly be accomplished by making a subset selection, and then providing a way to convert that into a SpectralRegion.  E.g., if the user uses the horizontal-selection to create a subset,  they can do:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reg = specviz.get_selected_region()  #<- uses the \"most recent\" selection\n",
```

Comment:
Hmm, I disagree because I think the one-level-up functionality returns *generic* regions, while the `specviz` one is specialized to return `SpectralRegions`.  I'll rename this more explicitly with that in mind, but if I'm misunderstanding feel free to point that out and we can post-merge update this.